tests/functional/modules/test_zos_job_query_func.py

```diff
@@ -23,6 +23,26 @@
 from shellescape import quote
 
 from ibm_zos_core.tests.helpers.dataset import get_tmp_ds_name
+from ibm_zos_core.tests.helpers.utils import get_random_file_name
+
+def get_job_id(hosts, len_id=9):
```

Comment:
Nice way of ensuring finding job ids without launching a new job. Will there be a limitation, like, what would happen in a recently IPLed system? I mean, I think it will continue with work with the system having STCs.

Reply:
There are jobs of IPL

Reply:
cool